examples/core/core_input_gamepad.nim

```diff
@@ -0,0 +1,179 @@
+#[
+*
+*   raylib [core] example - Gamepad input
+*
+*   NOTE: This example requires a Gamepad connected to the system
+*         raylib is configured to work with the following gamepads:
+*                - Xbox 360 Controller (Xbox 360, Xbox One)
+*                - PLAYSTATION(R)3 Controller
+*         Check raylib.h for buttons configuration
+*
+*   This example has been created using raylib 2.5 (www.raylib.com)
+*   raylib is licensed under an unmodified zlib/libpng license (View raylib.h for details)
+*
+*   Copyright (c) 2013-2019 Ramon Santamaria (@raysan5)
+*   Adapted in 2021 by jseb
+*
+********************************************************************************************]#
+
+import nimraylib_now
+
+# NOTE: Gamepad name ID depends on drivers and OS
+when defined(PLATFORM_RPI):
+  const XBOX360_NAME_ID = "Microsoft X-Box 360 pad"
+else:
+  const XBOX360_NAME_ID = "Xbox 360 Controller"
+
+const
+  PS3_NAME_ID = "PLAYSTATION(R)3 Controller"
+  XBOX360_LEGACY_NAME_ID = "Xbox Controller"
+  screenWidth = 800
+  screenHeight = 450
+
+type tex_names = enum PS3Pad, XboxPad
+var texs:array[tex_names,Texture2D]
```

Comment:
This array looks a bit unnecessary as we only use it for 2 textures. We could also make it closer to the original implementation so we have easier time updating it in the future

Reply:
Ok, it was for showing some things Nim can do, but of course for two variables it's not very useful.

When converting from C , the big question is always « should i stick to C primitivism or use features of my languages ? ». There are languages where you have no choices (convertion to C# or Java would involves at least a static class for  globals), and may be you don't want to lose your readers also. But for Nim, i believe showing some languages nice features can be interesting. Of course, it's harder to compare with the original, so let's keep the original for a public sample.

Reply:
Yes that is understandable. In this case I think it is a bit too much for my taste as it is not directly connected to raylib usage. It could be a bit overwhelming for newcomers to Nim to digest both this wrapper niceties and Nim-specific features, this is why I'd go with something simpler for this case